assemble/conf/accumulo.properties

```diff
@@ -31,3 +31,19 @@ instance.secret=DEFAULT
 
 ## Set to false if 'accumulo-util build-native' fails
 tserver.memory.maps.native.enabled=true
+
+## Set compaction settings
+### Default Compactor
+tserver.compaction.major.service.default.planner=org.apache.accumulo.core.spi.compaction.DefaultCompactionPlanner
```

Comment:
Could the new prefix be used here?

Reply:
It could be since the old rate.limit property was disassociated with the `default` compaction service.